README.md

```diff
@@ -37,12 +37,18 @@ MacVim.
 
 How do you do it?
 
-#### OS X 10.9 (Mavericks)
+#### OS X 10.9 (Mavericks) with boxen-web
 
 If you are using [`b26abd0` of boxen-web](https://github.com/boxen/boxen-web/commit/b26abd0d681129eba0b5f46ed43110d873d8fdc2)
-or newer, it will be automatically installed as part of Boxen.
+or newer, Xcode Command Line Tools will be automatically installed as part of Boxen.
 Otherwise, follow instructions below.
 
+##### OS X < 10.9 (Mavericks) without auto install
+
+1. Install Xcode from the Mac App Store.
+1. In terminal, run `xcode-select --install`
```

Comment:
I had to run this command to get postgres to compile earlier. Beforehand I was getting a warning, which I don't have shell history for anymore, that went away after running this. I found the fix here, https://github.com/Homebrew/homebrew/issues/21364#issuecomment-26941999